InventoryManagementSystem.py

```diff
@@ -0,0 +1,198 @@
+# Inventory Management System
+
+# Create a simple inventory management system for a small store. This problem will help you practice using classes, methods, loops, conditionals, and basic data structures.
+
+# Requirements:
+# 1. Create a `Product` class with the following attributes:
+#   -[x] name (string)
+#   -[x] price (float)
+#   -[x] quantity (int)
+
+class Product:
+    def __init__(self, name: str, price: float, quantity: int):
+        if price < 0:
+            raise ValueError("Price can't be negative!")
+        if quantity < 0:
+            raise ValueError("Quantity can't be negative!")
+        self.name = name
+        self.price = price
+        self.quantity = quantity
+
+    def get_value(self) -> float:
+        return self.price * self.quantity
+
+    def __repr__(self):
+        return f'{self.name}, {self.price}, {self.quantity}'
+    
+    @staticmethod
```

Comment:
What is this annotation for?

Reply:
Do you mean the @staticmethod? 
This is so I don't have to have the parsing logic within _load_inventory_ and is encapsulated within the Product class so if I need to modify anything it can be done from within there as well, as well as being able to easily reuse the _from_string_ elsewhere.

Reply:
Here is a bit of a better answer, make sure you understand fundamentally that the point of a static method is that it doesn't need to access attributes of the class object.

"I used @staticmethod because converting a string into a Product is an operation that doesn't need access to any instance data (doesn't need self). It's a standalone conversion utility that logically belongs to the Product class but doesn't need to interact with any existing Product object's attributes. This is different from methods like get_value() which need access to the instance's price and quantity through self."